src/storage.cpp

```diff
@@ -864,7 +864,7 @@ namespace libtorrent
 
 		// indices of all files we ended up copying. These need to be deleted
 		// later
-		aux::vector<bool, file_index_t> copied_files(f.num_files(), false);
+		aux::vector<bool, file_index_t> copied_files(std::size_t(f.num_files()), false);
```

Comment:
or maybe I argued against using the index type for size, or maybe the constructor isn't overridden